.editorconfig

```diff
@@ -9,4 +9,5 @@ trim_trailing_whitespace = true
 
 # JS/TS files
 [*.{js,jsx,ts,tsx}]
+end_of_line = lf
```

Comment:
Probably just move that to the `*` section as its enabled for all files in gittattributes

Reply:
From line 5:

> # (Please don't specify an indent_size here; that has too many unintended consequences.)

I don't disagree in principle, but don't want this PR to be the thing that could break a bunch.

Reply:
Ended-up doing that one in another PR as it was quite needed after the git change. I think you can remove that line now